e2e/testdata/ephemeral_test/pod-ephemeral-volume-mount.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: %s
+  namespace: %s
+spec:
+  containers:
+    - name: ubuntu
+      image: quay.io/cybozu/ubuntu:20.04
+      command: ["/usr/local/bin/pause"]
+      volumeMounts:
+        - mountPath: /test1
+          name: generic-ephemeral-volume
+  volumes:
+  - name: generic-ephemeral-volume
+    ephemeral:
+      volumeClaimTemplate:
+        metadata:
+            labels:
+              type: ubuntu-volume
+        spec:
+          accessModes:
+          - ReadWriteOnce
+          resources:
+            requests:
+              storage: 1Gi
```

Comment:
PVC Clone operations will fail because the size in the pvcCloneYAMLTemplate  does not match the size in the ephemeral volume spec. Please create a different yaml in the ephemeral_tests dir.